mibidata/color.py

```diff
@@ -0,0 +1,149 @@
+"""Color transformation and composites.
+
+Copyright (C) 2019 Ionpath, Inc.  All rights reserved."""
+
+import numpy as np
+
+COLORS = {
+    'Cyan': np.pi,
+    'Yellow': np.pi / 3,
+    'Magenta': 5 * np.pi / 3,
+    'Green': 2 * np.pi / 3,
+    'Orange': 0.2166 * np.pi,
+    'Violet': 1.483 * np.pi,
+    'Red': 0,
+    'Blue': 4 * np.pi / 3,
+}
+# Defines the minimum number of counts to scale images in
+# `composite`. This intended to prevent images with
+# only a few counts from being scaled to [0, 1].
+MIN_COUNTS_FOR_SCALING = 10
+
+
+def hsl2rgb(hsl):
```

Comment:
If you're going to pull in `hsl2rgb` here, you may as well pull in `rgb2hsl`. Better to have both in here and not need one than the other way around.

Reply:
Sure. I even threw in a couple more for free and added example usages the notebook.